website/docs/reference/define-configs.md

```diff
@@ -34,10 +34,14 @@ Configurations in your root dbt project have _higher_ precedence than configurat
 
 ## Combining configs
 
-Most configurations are "clobbered" when applied hierarchically. Whenever a more specific value is available, it will completely replace the less specific value. Note that a few configs have different merge behavior:
-- [`tags`](/tags) are additive. If a model has some tags configured in `dbt_project.yml`, and more tags applied in its `.sql` file, the final set of tags will include all of them.
-- [`meta`](/reference/resource-configs/meta) dictionaries are merged (a more specific key-value pair replaces a less specific value with the same key)
+Most configurations are "clobbered" or overwritten when applied hierarchically. Whenever a more specific value is available, it will completely replace the less specific value. Note that a few configs have different merge behavior:
```

Comment:
```suggestion
Most configurations are "clobbered," or overwritten in unexpected ways, when applied hierarchically. Whenever a more specific value is available, it will completely replace the less specific value. Note that a few configs have different merge behavior:
```